src/js/controllers/modals/search.js

```diff
@@ -77,4 +86,14 @@ angular.module('copayApp.controllers').controller('searchController', function($
     $scope.txHistoryShowMore = $scope.filteredTxHistory.length > $scope.txHistorySearchResults.length;
   };
 
+  $scope.searchOnBlockchain = function(searchTerm) {
+    var url = 'https://explorer.bitcoin.com/bch/search/' + searchTerm;
```

Comment:
The URL should cater to both BCH and BTC depending on whether the search was triggered from a BCH or BTC wallet.